Rakefile

```
@@ -25,7 +25,7 @@ end
 
 def use_asset_url_for_css_images
   path = "#{ASSETS_PATH}stylesheets/jsoneditor.scss"
-  modified_css = File.read(path).gsub(/url\("img\//, 'asset-url("')
+  modified_css = File.read(path).gsub(/url\("\.\/img\//, 'asset-url("')
```

What's the reason for this change? It was mentioned but not why. I'd prefer this remains as it was as a precaution. It's also how `asset-url` is documented by sass-rails.

```suggestion
  modified_css = File.read(path).gsub(/url\("img\//, 'asset-url("')
```

---

the reason is that the original code no longer works, as in it will not replace any instance of url() with asset-url, because the original code specifies that it needs to be `url("img)`, however with the new version there is no longer any instance of such string in the downloaded scss file anymore, because they are all `url("./img` now, so the regular expression cannot detect the parts that need to be changed into asset-url. you can try running the test without this change and only keeping the version change, it will fail because no code replacement actually happens. i hope that makes sense.

---

Yes, sorry, I misread the change initially. All good, thank you!